Cargo.toml

```
@@ -59,6 +59,9 @@ web_stub = ["webidl"]
     # [https://html.spec.whatwg.org/multipage/webstorage.html]
     webstorage = ["deno_webstorage", "webidl"]
 
+    # [https://websockets.spec.whatwg.org/]
+    websocket = ["deno_websocket", "deno_web"]
```

Oh wait hang on
This needs to depend on web not deno_web

```suggestion
    websocket = ["deno_websocket", "web"]
```

The web feature also lines up deno_web's dependencies